code/kthoom.js

```diff
@@ -692,6 +692,9 @@ class KthoomApp {
 
   showNextPage() {
     const turnedPage = this.bookViewer_.showNextPage();
+    if (this.bookViewer_.getFitMode() === FitMode.Width) {
```

Comment:
Can you also add a similar if to the showPrevPage() function?

Reply:
done in the last commit